association-backend/pyproject.toml

```diff
@@ -10,7 +10,7 @@ starlette = "^0.14.1"
 uvicorn = "^0.13.3"
 click = "7.1.2"
 invoke = "1.5.0"
-strawberry-graphql = "^0.57.1"
+strawberry-graphql = "^0.70"
```

Comment:
you need to update this on users-backend as well?

Reply:
Yes because was very behind and the `create_type` was release from `0.58.0` so I thought to just upgrade to the last available...

Reply:
Yeah sorry I meant that you need to update the other service as well as I don't see the change in users-backend. I am not sure how the tests even pass 🤔  maybe it already has the latest version?